paint/contracts/src/lib.rs

```diff
@@ -1034,6 +1034,7 @@ impl<T: Trait + Send + Sync> SignedExtension for CheckBlockGasLimit<T> {
 	type AccountId = T::AccountId;
 	type Call = <T as Trait>::Call;
 	type AdditionalSigned = ();
+	type Info = DispatchInfo;
```

Comment:
```suggestion
	type DispatchInfo = DispatchInfo;
```
I know `Info` is shorter to type, but it's really generic.

Reply:
fair enough 👍